demisto_sdk/commands/test_content/test_modeling_rule/tests/init_test_data_test.py

```diff
@@ -106,7 +107,9 @@ def test_init_test_data_create(pack):
     assert result.exit_code == 0
     assert test_data_file.exists() is True
     test_data = TestData.parse_file(test_data_file.as_posix())
+    event_fields: dict = test_data.data[0].expected_values or {}
```

Comment:
I don't understand the test here
you are asserting that 
` assert dict(sorted(test_data.data[0].expected_values)) == test_data.data[0].expected_values`
?